web/models/owners.ex

```diff
@@ -16,7 +16,7 @@ defmodule HexWeb.Owners do
       {:ok, _} ->
         owners = package |> all |> Users.with_emails
         owner = Enum.find(owners, &(&1.id == owner.id))
-        Mailer.send_owner_added_email(package, owners, owner)
+        Mail.owner_added(package, owners, owner) |> HexWeb.Mailer.deliver_now_throttled
```

Comment:
`HexWeb.Mailer.deliver_now_throttled` -> `Mailer.deliver_now_throttled` since we alias in `HexWeb.Web`